myst_nbs/time_series/Air_passengers-Prophet_with_Bayesian_workflow.myst.md

```diff
@@ -100,14 +107,17 @@ with pm.Model(check_bounds=False) as linear:
 fig, ax = plt.subplots(nrows=2, ncols=1, sharex=True)
 ax[0].plot(
     df["Month"],
```

Comment:
I would add the date from `df["Month"]` as a coordinate of the model, the name could be `month` or `date` or whatever you feel makes more sense.

```
coords = {"month": df["Month"]}
with pm.Model(coords=coords...
    trend = pm.Deterministic(..., dims="month")
    pm.Normal(..., dims="month")
```

I also have preference for using the name of the variable being predicted over "likelihood", I think here it could be "passengers" or "y" (to avoid confusing oneself and thinking it represents passenger count straight away)

Reply:
I also see there are a lot of `<model>_prior_predictive`, if you want to can name those `prior` only to make it a bit shorter (they have both groups) or use `trace` or `idata` and go on extending the same inferencedata as the workflow advances. 

I still haven't published so it looks a bit ugly right now but maybe it can already be helpful with the extension things, I am working on updating my pymc dims and coords blogpost to v4: https://github.com/OriolAbril/oriol_unraveled/blob/pymc_v4_arviz/_notebooks/2022-05-20-pymc-arviz.ipynb